public/js/map.js

```diff
@@ -255,12 +256,10 @@ exports.drawMap = (center, zoom, minZoom, draggable, handleMapClick, handleMarke
   createConstructionLayer();
 
   class1Layer.addTo(map);
-  class1LayerOutline.addTo(map);
   class2Layer.addTo(map);
   class3Layer.addTo(map);
   constructionLayer.addTo(map);
-  // Uncomment to add winterlayer by default:
-  // winterLayer.addTo(map);
+  winterLayer.addTo(map);
```

Comment:
You could do this programmatically by date if you know you always want it to show up on a certain date and disappear after a certain date.

Reply:
ooh good call